dev/scripts/load_test_customer_data.sh

```diff
@@ -44,8 +44,10 @@ NISE="$(which nise)"
 NISE_DATA_PATH="${DEV_SCRIPTS_PATH}/../../testing"
 
 # import common functions
-source $DEV_SCRIPTS_PATH/common/logging.sh
-source $DEV_SCRIPTS_PATH/common/utils.sh
+# shellcheck source=/dev/null
+source "$DEV_SCRIPTS_PATH"/common/logging.sh
+# shellcheck source=/dev/null
+source "$DEV_SCRIPTS_PATH"/common/utils.sh
```

Comment:
You can ignore  SC1090, SC1091, and SC2164.

Reply:
Just add that to `.shellcheckrc` either in the project on your your home directory. `shellcheck` just walks up the path to find the first settings file and uses that.